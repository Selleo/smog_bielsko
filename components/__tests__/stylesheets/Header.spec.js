import 'react-native';
import React                          from 'react';

import { shallow }                    from 'enzyme';
import chai, { expect }               from 'chai';
import dirtyChai                      from 'dirty-chai';

import Header                         from '../../stylesheets/Header';

const originalExpect = global.expect;

chai.use(dirtyChai);

describe('Header', () => {
  it('renders correctly', () => {
    originalExpect(Header).toMatchSnapshot();
    expect(Header).to.exist();
  });
});
