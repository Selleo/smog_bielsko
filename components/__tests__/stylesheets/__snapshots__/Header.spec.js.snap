exports[`Header renders correctly 1`] = `
ExtensibleStyleSheet {
  "$colorWhite": "white",
  "$headerHeight": 180,
  "_options": Object {
    "$colorWhite": "white",
    "$headerHeight": 180,
    "alignCenter": Object {
      "alignItems": "center",
    },
    "bgr": Object {
      "alignSelf": "stretch",
      "backgroundColor": "transparent",
      "borderWidth": 0,
      "flex": 1,
      "flexDirection": "column",
      "height": "$headerHeight",
      "justifyContent": "flex-end",
      "margin": 0,
      "padding": 0,
      "paddingBottom": 10,
      "top": 0,
      "width": undefined,
    },
    "bgr2": Object {
      "bottom": 0,
      "height": "$headerHeight",
      "left": 0,
      "position": "absolute",
      "resizeMode": "stretch",
      "right": 0,
      "top": -16,
    },
    "circle": Object {
      "alignItems": "center",
      "justifyContent": "center",
    },
    "circleContainer": Object {
      "alignItems": "center",
      "borderRadius": 30,
      "height": 15,
      "justifyContent": "center",
      "overflow": "hidden",
      "width": 15,
    },
    "container": Object {
      "height": "$headerHeight",
      "position": "relative",
      "width": undefined,
    },
    "description": Object {
      "backgroundColor": "transparent",
      "color": "$colorWhite",
      "marginLeft": 20,
      "paddingTop": 3,
      "position": "relative",
      "zIndex": 1,
    },
    "elevatedView": Object {
      "alignItems": "stretch",
      "backgroundColor": "white",
      "justifyContent": "space-between",
      "margin": 20,
      "marginBottom": 15,
      "marginTop": 0,
    },
    "spaceBetweenItems": Object {
      "paddingTop": 5,
    },
    "title": Object {
      "backgroundColor": "transparent",
      "color": "$colorWhite",
      "fontSize": 30,
      "marginLeft": 20,
      "marginTop": 90,
      "position": "relative",
      "zIndex": 1,
    },
    "withShadow": Object {
      "alignItems": "stretch",
      "backgroundColor": "white",
      "borderRadius": 3,
    },
    "withShadow2": Object {
      "backgroundColor": "white",
      "borderRadius": 3,
    },
  },
  "_styles": Object {
    "alignCenter": Object {
      "alignItems": "center",
    },
    "bgr": Object {
      "alignSelf": "stretch",
      "backgroundColor": "transparent",
      "borderWidth": 0,
      "flex": 1,
      "flexDirection": "column",
      "height": 180,
      "justifyContent": "flex-end",
      "margin": 0,
      "padding": 0,
      "paddingBottom": 10,
      "top": 0,
      "width": undefined,
    },
    "bgr2": Object {
      "bottom": 0,
      "height": 180,
      "left": 0,
      "position": "absolute",
      "resizeMode": "stretch",
      "right": 0,
      "top": -16,
    },
    "circle": Object {
      "alignItems": "center",
      "justifyContent": "center",
    },
    "circleContainer": Object {
      "alignItems": "center",
      "borderRadius": 30,
      "height": 15,
      "justifyContent": "center",
      "overflow": "hidden",
      "width": 15,
    },
    "container": Object {
      "height": 180,
      "position": "relative",
      "width": undefined,
    },
    "description": Object {
      "backgroundColor": "transparent",
      "color": "white",
      "marginLeft": 20,
      "paddingTop": 3,
      "position": "relative",
      "zIndex": 1,
    },
    "elevatedView": Object {
      "alignItems": "stretch",
      "backgroundColor": "white",
      "justifyContent": "space-between",
      "margin": 20,
      "marginBottom": 15,
      "marginTop": 0,
    },
    "spaceBetweenItems": Object {
      "paddingTop": 5,
    },
    "title": Object {
      "backgroundColor": "transparent",
      "color": "white",
      "fontSize": 30,
      "marginLeft": 20,
      "marginTop": 90,
      "position": "relative",
      "zIndex": 1,
    },
    "withShadow": Object {
      "alignItems": "stretch",
      "backgroundColor": "white",
      "borderRadius": 3,
    },
    "withShadow2": Object {
      "backgroundColor": "white",
      "borderRadius": 3,
    },
  },
  "_variables": Object {
    "$colorWhite": "white",
    "$headerHeight": 180,
  },
  "alignCenter": Object {
    "alignItems": "center",
  },
  "bgr": Object {
    "alignSelf": "stretch",
    "backgroundColor": "transparent",
    "borderWidth": 0,
    "flex": 1,
    "flexDirection": "column",
    "height": 180,
    "justifyContent": "flex-end",
    "margin": 0,
    "padding": 0,
    "paddingBottom": 10,
    "top": 0,
    "width": undefined,
  },
  "bgr2": Object {
    "bottom": 0,
    "height": 180,
    "left": 0,
    "position": "absolute",
    "resizeMode": "stretch",
    "right": 0,
    "top": -16,
  },
  "circle": Object {
    "alignItems": "center",
    "justifyContent": "center",
  },
  "circleContainer": Object {
    "alignItems": "center",
    "borderRadius": 30,
    "height": 15,
    "justifyContent": "center",
    "overflow": "hidden",
    "width": 15,
  },
  "container": Object {
    "height": 180,
    "position": "relative",
    "width": undefined,
  },
  "description": Object {
    "backgroundColor": "transparent",
    "color": "white",
    "marginLeft": 20,
    "paddingTop": 3,
    "position": "relative",
    "zIndex": 1,
  },
  "elevatedView": Object {
    "alignItems": "stretch",
    "backgroundColor": "white",
    "justifyContent": "space-between",
    "margin": 20,
    "marginBottom": 15,
    "marginTop": 0,
  },
  "spaceBetweenItems": Object {
    "paddingTop": 5,
  },
  "title": Object {
    "backgroundColor": "transparent",
    "color": "white",
    "fontSize": 30,
    "marginLeft": 20,
    "marginTop": 90,
    "position": "relative",
    "zIndex": 1,
  },
  "withShadow": Object {
    "alignItems": "stretch",
    "backgroundColor": "white",
    "borderRadius": 3,
  },
  "withShadow2": Object {
    "backgroundColor": "white",
    "borderRadius": 3,
  },
}
`;
